int16[] ids
geometry_msgs/PolygonStamped[] hulls

